CHANGELOG.md

```diff
@@ -1,2 +1,8 @@
+- Adds support for the Firestore Provisioning API commands: (#5616)
```

Comment:
Consider rewording this message to be a little clearer to external users - ie "Adds new commands for provisioning and managing Firestore databases:"

Also, include the databases:update command here too